R/cohens_g.R

```diff
@@ -35,13 +35,22 @@
 #' - Cohen, J. (1988). Statistical power analysis for the behavioral sciences (2nd Ed.). New York: Routledge.
 #'
 #' @examples
-#' Performance <- matrix(c(794, 150,
-#'                         86, 570), nrow = 2)
-#' dimnames(Performance) <- list("1st Survey" = c("Approve", "Disapprove"),
-#'                               "2nd Survey" = c("Approve", "Disapprove"))
-#' Performance
 #'
-#' cohens_g(Performance)
+#' data("Test_results")
```

Comment:
_Test_results_ is too generic.

Can you think of a more descriptive name? How about _screening_test_ or just _screening_?